README.md

```diff
@@ -965,15 +965,21 @@ not persisted between runs.
 
 ## Parser cache
 
-A cache of parsed Puppet modules and manifests can be created to skip the use
+One or more caches of parsed Puppet modules and manifests can be created to skip the use
```

Comment:
Ah sorry, github didn't refresh and I missed this comment, I'll send a separate PR with more info.